sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -47,8 +47,12 @@ def healthcheck(request):
 
 def aws_sdk_call(request):
     bucket_name = "e2e-test-bucket-name"
-    s3_client = boto3.client("s3")
 
+    # Add an (pod) ID to bucketname to associate buckets to specific test runs
+    ip = request.GET.get('ip', None)
+    if ip is not None:
+        bucket_name += "-" + ip
```

Comment:
Are we going to create buckets with ip suffix in all regions/accounts? Otherwise, there will be error message with bucket not found.

Reply:
The buckets are not expected to exist (with and without the ip identifier)

See comment: https://github.com/aws-observability/aws-application-signals-test-framework/blob/main/sample-apps/springboot/src/main/java/com/amazon/sampleapp/FrontendServiceController.java#L89